### ISSUE TITLE
  Sprint 2 – Scrum Meeting (10-12-2016) 

### ISSUE THREAD

**Issue by huylin25** [mewind/RSDSupermarketPOS#11]
Hi guys, we have only 2 days left before ending the Sprint 2, do you guys have anymore problem when developing the functions?

**Comment by mewind** [comment]
In my part, actually I've done the Search function today before the schedule, and I'll be uploading the code latest by tomorrow.

**Comment by huylin25** [comment]
Wow! So efficient! So do you mean that you've completed all the tasks given ?

**Comment by mewind** [comment]
Ermmmm, I think yes?

**Comment by huylin25** [comment]
So can you briefly explain what you guys included in the Search function and how it works?

**Comment by lkc442012** [comment]
I think I'll try to explain this. Actually after finishing the Add To Cart function which is the hardest part, the remaining function is quite easy to develop. For the Search function, we just add a textbox and a dropdown list to let the user enter the product name and choose the category in order to filter the searching result.

**Comment by huylin25** [comment]
So, you have developed the code behind to make the function worked?

**Comment by lkc442012** [comment]
Yes, we had already tested the function and make sure they are fully working

**Comment by huylin25** [comment]
Great! Raymond how about you?

**Comment by juzplayplay** [comment]
Actually I just done my part and try to upload the code latest by tomorrow too, for me I'll just check the product user selected whether have sufficient stock, if the quantity user wish to buy exceeded the stock quantity, the system will return an error message to the user

**Comment by huylin25** [comment]
Seems you guys have no problem with both of these function, so I think we will stop here and you guys can have a rest. We will conclude what we have done after inspecting the full outcome of this week tomorrow. Thanks guys!

**Comment by lkc442012** [comment]
Thanks Huy Lin!

**Comment by mewind** [comment]
ThankYou!

**Comment by juzplayplay** [comment]
Thank you Huy Lin.
